src/js/Autocompletes/Autocomplete.d.ts

```diff
@@ -31,19 +32,22 @@ export interface AutocompleteProps extends BaseMenuProps {
   total?: number;
   offset?: number;
   filter?: (data: DataType, filterText: string | number, dataLabel?: string) => Array<string>;
-  fullWidth?: boolean;
   inline?: boolean;
   findInlineSuggestion?: (data: DataType, value: string | number, dataLabel?: string) => string | number;
   autocompleteWithLabel?: boolean;
   onAutocomplete?: (suggestion: string | number, suggestionIndex: number, matches: DataType) => void;
+  onBlur?: (event: React.FocusEvent<HTMLFormElement>) => void,
```

Comment:
The `onBlur`, `onFocus` and `onKeyDown` props _should_ be redundant since they are inherited from `Props` (-> `SharedLayoverProps` -> `BaseMenuProps`).

It also looks like there commas instead of semi-colons for these two which broke the build:
```
lib/Autocompletes/Autocomplete.d.ts(42,69): error TS1131: Property or signature expected.
```

Would you mind fixing that and make the `toolbar` prop optional?